tools/toolchains/iar.py

```diff
@@ -186,6 +195,10 @@ def archive(self, objects, lib_path):
 
         self.default_cmd([self.ar, lib_path, '-f', archive_files])
 
+    def link(self, output, objects, libraries, lib_dirs, mem_map):
```

Comment:
there's hook above taht needs a fix (add flags['ld']), that should be fixed. I've got a local patch, cant push to your remote, thus will create new PR